themes/cantera/templates/jupyter-example-index.tmpl

```diff
@@ -6,16 +6,21 @@
 <p>Cantera examples in the form of <a href="https://jupyter.org" title="Jupyter">Jupyter</a>
 notebooks. To see the rendered notebooks, browse the files below.</p>
 
-<p><strong>Existing Cantera users</strong>: If you have Cantera and the Jupyter Notebook server
-installed on your local machine, simply download any Jupyter notebook by clicking the "Source" link
-at the top of each example page and you should be able to run it.</p>
+<p><strong>Existing Cantera users</strong>: If you have Cantera and the Jupyter Notebook
+server installed on your local machine, simply download any Jupyter notebook by clicking
+the "Source" link at the top of each example page and you should be able to run it. For
+some examples, you will also need to download the additional data files listed below the
+"Source" link. These data files should either be placed in a subdirectory named "data"
+one level up from the directory containing the notebook file, or the notebook can be
+modified to point to the location of these data files on your computer.
```

Comment:
On second thought: would it make sense to just tell users to clone the `cantera-jupyter` repo?

Reply:
No, I don't think we want to put the barrier of using Git between our users and some of our most readily accessible examples.

Reply:
Ok. In this case I’m ok with merging … :shipit:

Reply:
Well, on second thought, I don't think it hurts to mention the option of cloning the repo as an alternative.